.github/pull_request_template.md

```diff
@@ -1,6 +1,6 @@
 ### Pull Request Checklist
 
-- [ ] Read and understand the [contribution guidelines](./CONTRIBUTING.md)
+- [ ] Read and understand the [contribution guidelines](https://github.com/digital-asset/daml/blob/master/CONTRIBUTING.md)
```

Comment:
Ah, good catch, thanks! Since you're there, would you mind also fixing the relative path in the very last item? After that we'd be good to go.